pontoon/api/urls.py

```diff
@@ -4,8 +4,8 @@
 from django.conf.urls import url
 from django.views.decorators.csrf import csrf_exempt
 
+from pontoon.settings import DEV
```

Comment:
``api`` comes before ``settings`` in my alphabet. ;-)

Reply:
Ah, sorry, I thought more general module paths should go first.